src/LocalFile.js

```diff
@@ -1,77 +1,85 @@
-const { createRemoteFileNode } = require('gatsby-source-filesystem');
+const { createRemoteFileNode } = require('gatsby-source-filesystem')
 
 class LocalFile {
   constructor(createRemoteArgs) {
-    this.createRemoteArgs = createRemoteArgs;
```

Comment:
prettier would help keep PR diffs smaller

Reply:
Woops, looks like my default prettier config kicked in. Maybe it would be a good idea to add prettier/eslint configs to the repo @njosefbeck?

Reply:
I'm cool with that! Actually, I'll just give you contributor access @brxck so you can set that up. I'm unfortunately a little pressed for time at the moment. Just make sure you add these changes + the edition of prettier / eslint to the CHANGELOG. Once everything is in master as we like it, I can handle publishing it to npm.

LMK if that works for you!